modules/cockroachdb/options.go

```diff
@@ -3,11 +3,12 @@ package cockroachdb
 import "github.com/testcontainers/testcontainers-go"
 
 type options struct {
-	Database  string
-	User      string
-	Password  string
-	StoreSize string
-	TLS       *TLSConfig
+	Database   string
+	User       string
+	Password   string
+	StoreSize  string
+	TLS        *TLSConfig
+	Statements []string
```

Comment:
question: As we have this should we set the default so the cluster is setup with best practices?

Reply:
makes sense to me. It is a bit weird to require the user to have to pass `cockroachdb.WithStatements()` to disable the defaults though.

Reply:
Hmm, yer @mdelapenya what do you recon?

Reply:
Yes, default statements should include the recommended defaults from the vendor

Reply:
Indeed, thanks for considering it too.

Reply:
Sounds sensible to me